platform_plugin_communications/apps.py

```diff
@@ -4,10 +4,32 @@
 
 from django.apps import AppConfig
 
+try:
+    from openedx.core.constants import COURSE_ID_PATTERN
+except ImportError:
+    COURSE_ID_PATTERN = object()
+
 
 class PlatformPluginCommunicationsConfig(AppConfig):
     """
     Configuration for the platform_plugin_communications Django application.
     """
 
     name = "platform_plugin_communications"
+
+    plugin_app = {
+        "url_config": {
+            "lms.djangoapp": {
+                "namespace": "platform_plugin_communications",
+                "regex": rf"courses/{COURSE_ID_PATTERN}/instructor/api/",
```

Comment:
No, we are using the same regex and it wil lbe used as prefix of the url patterns

Reply:
As I see in the testing example: `/courses/course-v1:demo+demo+demo/instructor/api`, we're using the same path and we shouldn't since it's a different API.

Reply:
Ok, so what should be the path?

Reply:
What about: 
/platform-plugin-communications/instructor-emails/<COURSE_ID>
Or something like it